src/FetchRequestPerformer.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+import type { RequestPerformer } from './types';
+
+class FetchRequestPerformer implements RequestPerformer<Request, Response> {
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
This could likely be resolved by writing this method as `performRequest = fetch;`